Exercise_9.R

```diff
@@ -0,0 +1,210 @@
+#Question 1
+
+library(ggplot2)
+
+ggplot(data)+
+  geom_point(aes(x=mutation, y=ponzr1Counts))
+
+ggplot(data)+
+  geom_boxplot(aes(x=mutation, y=ponzr1Counts))
+
+data <- read.csv("/Users/elizabethfortin12/Documents/ND First Year/Biocomputing/R_Programming/Exercise_9/ponzr1.csv")
+
+# adding a column for mutated = 1, wt = 0
+for (i in 1:nrow(data)){
+  if (data$mutation[i] == 'WT'){
+    data$mutorno[i] <- 0
+  }
+  else{
+    data$mutorno[i] <- 1
+  }
+}
+
+# data frames containing only one mutation
+m124k <- data[data$mutation=='WT',]
+m124k[11:20,] <- data[data$mutation=='M124K',]
```

Comment:
Does this work? I learned something new! Could do data[data$mutation==c('WT','M124K'),]